src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/document/DocumentStoreImpl.java

```diff
@@ -17,4 +25,9 @@
         return null;
     }
 
+    @Override
+    @Cacheable(cacheNames = "documentDetails", cacheManager = "documentDetailsCacheManager", unless = "#result == null")
```

Comment:
question: what does `unless = #result == null` do?
is it just to prevent caching null documents?

Reply:
yes it does, otherwise would put null values in the cache